### PR TITLE
Match card component design

### PR DESCRIPTION
**Pull request by sdlyy**
Resolves L2B-3045

### CODE REVIEW

File: packages/frontend/src/view/components/RowSeparator.tsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export function RowSeparator() {
+  return <div className="h-px w-full bg-gray-400" />
```
Comment (michalsidzej) [inline packages/frontend/src/view/components/RowSeparator.tsx:2]:
```suggestion
  return <hr className="h-px w-full bg-gray-400" />
```

File: packages/frontend/src/view/components/protocol/EndpointContract.tsx [hunk 2]
```diff
@@ -14,12 +17,21 @@ interface Props {
...
-        label="Default receive library"
-        value={props.defaultReceiveLibrary}
-      />
+      <ExpandableContainer
+        showText="View contract parameters"
+        hideText="Hide contract parameters"
+      >
+        <Subsection>
+          <Row label="Owner" value={props.owner} />
+          <RowSeparator />
```
Comment (michalsidzej) [inline packages/frontend/src/view/components/protocol/EndpointContract.tsx:26]: Row separator could be a border of every non last row (nitpick)

**Review by michalsidzej — APPROVED** [review]
Small comments, but LGTM

File: packages/frontend/src/view/components/protocol/EndpointContract.tsx (hunk 2 above)
Comment (sdlyy) [reply packages/frontend/src/view/components/protocol/EndpointContract.tsx]: [feat: remove row separator](https://github.com/l2beat/lz-monitoring/pull/68/commits/7bf7c42282c2a7bd23211b68135ea4e38ce1a582)

**Pull request merged**